pixi.toml

```diff
@@ -0,0 +1,11 @@
+[workspace]
+authors = ["Lena Poepping <80910285+Zeyna777@users.noreply.github.com>"]
+channels = ["conda-forge"]
+name = "k-matrix-day"
+platforms = ["linux-64"]
+version = "0.1.0"
+
+[tasks]
```

Comment:
Here you can define job names for simplifying the build. E.g.
```toml
[tasks.doc]
cmd = "quarto render docs"

[tasks.doclive]
cmd = "quarto preview docs --port 7000"
```
so that you can simply run
```shel
pixi run doc
```
without having to activate the environment.